openapi.yaml

```diff
@@ -13797,6 +13797,8 @@ components:
           description: >
             The relative weight of this Record. Higher values are preferred.
           example: 50
+          minimum: 0
+          maximum: 65535
           x-linode-cli-display: 7
         port:
           type: integer
```

Comment:
While we didn't change it as part of this ticket, the port min/max is the same as the weight min/max. Can we add that here?

Reply:
sure, will do

Reply:
@alsaihn added in 14cb3b7